all-clouds/all-clouds-explanation/cloud-init-metapackages.rst

```diff
@@ -0,0 +1,21 @@
+Cloud-init metapackages
+=======================
+
+Starting in Ubuntu Oracular (24.10), `cloud-init <https://launchpad.net/ubuntu/+source/cloud-init>`_
```

Comment:
I think this link should just point to the main cloud-init documentation (rather than the code), since it is in the introductory paragraph and one tends to assume that the link will tell me what cloud-init is.

We should mention the source package link somewhere separately to indicate it as the location for finding the metapackages. Although, even in that link, the specific metapackages available are not obvious. So some detail about how to find them might be useful. - Maybe create a small section at the end called 'Installation guidelines' and add these details there.

Reply:
> I think this link should just point to the main cloud-init documentation (rather than the code), since it is in the introductory paragraph and one tends to assume that the link will tell me what cloud-init is.

> We should mention the source package link somewhere separately to indicate it as the location for finding the metapackages.

Done, thanks for the suggestion.

> Although, even in that link, the specific metapackages available are not obvious. So some detail about how to find them might be useful. - Maybe create a small section at the end called 'Installation guidelines' and add these details there.

Those packages can be found in the cloud-init source package link and find / installed using `apt-get`, `apt` and the like. But, being this page targeting Ubuntu image creators / builders, I believe your suggesting would probably be over-documenting. WDYT?

Reply:
Yeah I agree, the way it is written now should be enough.

Though I had assumed those packages could be installed using apt, the package names were not evident in the source link. But we are specifying the package names in this page, so it's fine.